From 7b9d25ee79939e4c22a82b24cd42c4671fb843f0 Mon Sep 17 00:00:00 2001
From: Gioele Barabucci <gioele@svario.it>
Date: Mon, 14 Mar 2022 18:20:46 +0100
Subject: [PATCH 1/2] debian/current-md5sum-obsolete: Remove unused script

The script `current-md5sum-obsolete` is no longer needed (nor functional)
since Debian 7 wheezy (2016).
---
 debian/current-md5sums-obsolete | 3 ---
 1 file changed, 3 deletions(-)
 delete mode 100755 debian/current-md5sums-obsolete

diff --git a/debian/current-md5sums-obsolete b/debian/current-md5sums-obsolete
deleted file mode 100755
index 5236dc4..0000000
--- a/debian/current-md5sums-obsolete
+++ /dev/null
@@ -1,3 +0,0 @@
-#!/bin/sh
-sed -e "s&#OSNAME#&GNU/Linux&" etc/motd | awk 'NR > 2' | md5sum
-sed -e "s&#OSNAME#&GNU/Hurd&"  etc/motd | awk 'NR > 2' | md5sum
-- 
2.34.1

